app/gui/language/ast/impl/src/opr.rs

```diff
@@ -260,18 +260,28 @@ impl GeneralizedInfix {
     }
 
     /// The self operand, target of the application.
-    pub fn target_operand(&self) -> Operand {
+    pub fn target_operand(&self) -> &Operand {
         match self.assoc() {
-            Assoc::Left => self.left.clone(),
-            Assoc::Right => self.right.clone(),
+            Assoc::Left => &self.left,
+            Assoc::Right => &self.right,
         }
     }
 
     /// Operand other than self.
-    pub fn argument_operand(&self) -> Operand {
+    pub fn argument_operand(&self) -> &Operand {
         match self.assoc() {
-            Assoc::Left => self.right.clone(),
-            Assoc::Right => self.left.clone(),
+            Assoc::Left => &self.right,
+            Assoc::Right => &self.left,
+        }
+    }
+
+    /// Get a relative AST crumb pointing to the argument operand.
```

Comment:
I dont understand this sentence. Could you make it a little bit more descriptive, please?